pkg/process/config/config.go

```diff
@@ -151,6 +151,12 @@ func NewDefaultAgentConfig() *AgentConfig {
 	//       need a few minutes to be ready.
 	_, err = util.GetContainers()
 	canAccessContainers := err == nil
+	canAccessContainers = false
+
+	enabledChecks := make([]string, 0)
```

Comment:
Should probably be `var enabledChecks []string` unless you mean to allocate an empty array rather than a nil slice.

Reply:
It can be nil.